packages/mdx/package.json

```
@@ -47,6 +47,8 @@
   ],
   "dependencies": {
     "@types/estree-jsx": "^1.0.0",
+    "@types/estree": "^1.0.0",
```

Could you sort these alphabetically??

```suggestion
    "@types/estree": "^1.0.0",
    "@types/estree-jsx": "^1.0.0",
```

---

Of course 👍